virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -129,7 +127,8 @@ Removed features are not supported in the current release.
 
 //NOTE: RNs related to 4.14 Removed features begin here.
 
-//CNV-27160 Release note: REMOVED (could be dupe of CNV-28717; asked Avital)
+//CNV-27160 Release note: REMOVED RHEL 7/virtctl RPMs 
+* {op-system-base-full} 7 and {op-system-base} 9 are no longer supported for `virtctl` command line tool installation by an RPM.
```

Comment:
For the most part, this looks good to me. I just want to verify that the use of "supported" is accurate here. The way the sentence is worded makes it sound like the cl tool installation no longer supports RHEL 7 or 9. But do you mean that RHEL 7 and 9 no longer support this sort of tool installation? For example:

```suggestion
* {op-system-base-full} 7 and {op-system-base} 9 no longer support the installation of the `virtctl` command-line tool by an RPM.
```

Or

```suggestion
* The installation of the `virtctl` command-line tool by an RPM no longer supports {op-system-base-full} 7 and {op-system-base} 9.
```

Reply:
@sjhala-ccs

Reply:
I agree with @DCChadwick's suggestion to rephrase this sentence for clarity. Would something like this make more sense:
```suggestion
* Installing the `virtctl` client as an RPM is no longer supported for {op-system-base-full} 7 and {op-system-base} 9.
```

Reply:
@DCChadwick @sjhala-ccs  I agree, too! Thanks to you both.